.github/workflows/clean-up-closed-prs.yml

```
@@ -53,3 +53,23 @@ jobs:
         run: |
           echo "::notice ::Cancelling tests on closed PR #$PR"
           gh run cancel "$RUN_ID"
+
+  delete-branch:
+    if: >
+      github.repository_owner == 'Homebrew' &&
+      !github.event.pull_request.merged
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    continue-on-error: true
```
This seems more idiomatic for what you want to do here with `|| true`.

---

Thanks; doing this in #177336